lib/DDG/Spice/Timer.pm

```diff
@@ -13,12 +22,19 @@ handle remainder => sub {
     my $qry = $_;
     my $raw = lc($req->query_raw);
     my $trgx = join('|', @triggers);
-
+    my $stTrgx = join('|', @startTringgers);
```

Comment:
@vCillusion we should be defining all these static regexes outside the `handle` so they're only created once and stored in memory.

This way, we're building all these regexes each time the IA triggers!